cloud-common/build.gradle

```diff
@@ -20,4 +20,5 @@ dependencies {
     compile group: 'commons-io', name: 'commons-io', version: '2.4'
     compile group: 'com.google.guava', name: 'guava', version: guavaVersion
     compile group: 'org.freemarker',                name: 'freemarker',                     version: freemarkerVersion
+    runtime('org.springframework.cloud:spring-cloud-starter-consul-config:1.0.0.BUILD-SNAPSHOT')
```

Comment:
please use the correct dependency definition format:
runtime group: x, name: y, version: z